### PR TITLE
Feature/issue #93

### PR DESCRIPTION
**Pull request by amba178**
I didn't understand when you testing... Do you want me to write test for views.There are from devise.

### CODE REVIEW

File: app/views/devise/registrations/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+  = devise_error_messages!
+  .field
+    = f.label :email
+    %br/
+    = f.email_field :email, autofocus: true
+  .field
+    = f.label :first_name
+    %br/
+    = f.email_field :first_name
```
Comment (joaopapereira) [inline app/views/devise/registrations/new.html.haml]: This is incorrect

File: app/views/devise/registrations/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  .field
+    = f.label :last_name
+    %br/
+    = f.email_field :last_name
```
Comment (joaopapereira) [inline app/views/devise/registrations/new.html.haml]: This is incorrect, is pointing to a email field

File: app/views/devise/registrations/new.html.haml (hunk 1 above)
Comment (amba178) [inline app/views/devise/registrations/new.html.haml]: Yes, that is original error. initial was there. I hopefully Emy took care of it.

File: app/views/devise/registrations/new.html.haml (hunk 2 above)
Comment (amba178) [inline app/views/devise/registrations/new.html.haml]: The same thing here. Emy version should fix that. I did not want to create conflict by changing the fields error you talking about.

**Comment by amba178** [comment]
Will create new PR.

**Pull request closed without merging**